app/views/promotion/weeklyLandingPage.scala.html

```diff
@@ -2,94 +2,133 @@
 @import com.gu.memsub.promo.Promotion.PromoWithWeeklyLandingPage
 @import com.gu.memsub.promo.Promotion.asAnyPromotion
 @import views.support.LandingPageOps._
+@import configuration.Links
+@import views.support.CommaSplit
 @import views.support.{MarkdownRenderer}
 @import com.gu.memsub.subsv2.Catalog
 @import model.Subscriptions._
+@import controllers.CachedAssets.hashedPathFor
 @import com.gu.i18n.Country
 @import views.support.WeeklyPromotion
-@(country: Country, catalog: Catalog, promoCode: PromoCode, promotion: PromoWithWeeklyLandingPage, md: MarkdownRenderer)
-    @voucherPlans = @{catalog.voucher.list.filter(p => promotion.appliesTo.productRatePlanIds.contains(p.id))}
-    @deliveryPlans = @{catalog.delivery.list.filter(p => promotion.appliesTo.productRatePlanIds.contains(p.id))}
-    @anyPromotion = @{asAnyPromotion(promotion)}
-    @title = @{"Subscribe to the Guardian Weekly"}
-
+@import com.gu.memsub.promo.Grey
+@import com.gu.memsub.promo.Blue
+@import com.gu.memsub.promo.White
+@(country: Country, catalog: Catalog, promoCode: Option[PromoCode], promotion: Option[PromoWithWeeklyLandingPage],description:Html, md: MarkdownRenderer)
+    @anyPromotion = @{promotion.map(asAnyPromotion)}
+    @nonTrackingPromotion = @{anyPromotion.flatMap(p => p.asDiscount orElse p.asFreeTrial orElse p.asIncentive)}
+    @title = @{"The Guardian Weekly"}
     @discountedRegions = @{WeeklyPromotion.validRegionsForPromotion(promotion, promoCode, country)(catalog)}
+    @main(title = s"$title | The Guardian", bodyClasses = List("is-wide")) {
+    @promotion.flatMap(_.landingPage.image).fold{
+        @*
+        don't ship this, use a grid image
+        *@
+        <div class="hero-banner">
+            <div class="hero-banner__underlay gs-container"></div>
+            <picture class="hero-banner__image">
+                <img src="@hashedPathFor("images/backgrounds/weeklydontuse.jpg")" sizes="100vw"/>
+            </picture>
 
-    @main(title = s"$title | The Guardian") {
-        <main class="page-container gs-container">
-            @fragments.page.header(title, promotion.landingPage.title)
-            @promotion.landingPage.roundelHtml.map { h =>
-                <div class="digipack__roundel">@fragments.common.roundel(h, Seq("roundel--raised", "roundel--right"))</div>
-            }
-            @promotion.landingPage.image.map { img =>
-                <img src="@img.defaultImage" srcset="@img.srcset" sizes="100vw" class="promotion-image__image"/>
+            <div class="hero-banner__inner idk-flexwrap ">
+
+                <div class="hero-banner__content section-left">
+                    <h1 class="hero-banner__title">@title</h1>
+                    <h1 class="hero-banner__description">@CommaSplit.split(promotion.flatMap(_.landingPage.title).getOrElse("Get the world in perspective,\nevery single week"))</h1>
```

Comment:
Nice solution. Also you don't need the \n in the or-else text now that you have a comma split.